### DISCUSSION TITLE
Q-Table, buttons at **end** of rows.

### DISCUSSION THREAD

**Discussion by mtn090899** [quasarframework/quasar#12992] [General - Components / Directives / etc]
Hi,

I recently had another question (maybe a couple days ago) and I figured that piece out! Was looking at it all wrong but figured it out. Now, I want to add buttons at the **end** of each row on q-table. I was able to do so with v-slot:body="props", but what fixed my previous issue was just using v-slot:body-election="scope".

Unfortunately, now I do not know how to add any buttons/toggles at the end of rows. I've tried utilizing CSS, but I've had no luck with that. 

Here is my code, any advice would be great!

```<template>
  <q-table
    :title="title"
    :rows="rows"
    :columns="columns"
    :row-key="row_key"
    :table-header-style="{ backgroundColor: '#DCEEFB' }"
    :options="options"
    
  >
      <template v-slot:header-selection="scope">
        <q-btn-dropdown flat class="something" disable no-icon-animation dropdown-icon=" " />
        <q-checkbox v-model="scope.selected" />
      </template>

        <template v-slot:body-selection="scope">
            <q-btn-dropdown flat class="something" no-icon-animation dropdown-icon="las la-ellipsis-v" />
            <q-checkbox v-model="scope.selected" />
        
      </template>
   
  </q-table>
</template>

<script>
export default {
  data() {
    return {
      slot_drop: false,
    };
  },
  props: {
    columns: {
      type: Array,
    },
    rows: {
      type: Array,
    },
    title: {
      type: String,
    },
    row_key: {
      type: String,
    },
    options: {
      type: Boolean,
      default: false,
    },
  },

};
</script>

<style>
.q-table__sort-icon {
  opacity: 1;
}

</style>
```

Screen-Cap:
![image](https://user-images.githubusercontent.com/78068147/160892626-0b8977f7-9bf5-40e1-8095-4c4f7c1c18e6.png)

**Comment by metalsadman** [comment]
first table https://codepen.io/metalsadman/pen/ZgKexK
